test/extended/cli/status.go

```diff
@@ -85,9 +85,11 @@ var _ = g.Describe("[sig-cli] oc status", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		g.By("verify jobs are showing in status")
-		err = oc.WithoutNamespace().Run("create").Args("job", "pi", "--image=image-registry.openshift-image-registry.svc:5000/openshift/tools:latest",
-			"--", "perl", "-Mbignum=bpi", "-wle", "'print bpi(2000)'").Execute()
+		err = oc.WithoutNamespace().Run("create").Args("job", "pi", "--image=image-registry.openshift-image-registry.svc:5000/openshift/tools:latest").Execute()
```

Comment:
As long as `WithoutNamespace` makes sense in the above calls, here when you're creating namespaced resources I'd suggest creating jobs also in a particular namespace, and also checking status in that same namespace. Also, you should be cleaning all the projects being created in this test, which will handle all the resources inside them as well.